enterprise-suite/README.md

```diff
@@ -69,17 +69,11 @@ These assume you're using minikube.
 
 ## Cutting a Release / Publishing Charts
 
-_Is this still correct?_
-
 ### Release ES images
 
-Release the images in Jenkins:
-1. Go to <https://ci.lightbend.com/view/EntSuite/job/es-release-all/build>.
-2. Click 'Build'. This will increment the version for each image and release it.
-3. Open the [console](https://ci.lightbend.com/view/EntSuite/job/es-release-all/lastBuild/console), scroll to the bottom and you can see the versions of each image.
-4. Update [enterprise-suite/values.yaml](enterprise-suite/values.yaml) with the new image versions and commit the changes.
-
-See [es-build](https://github.com/lightbend/es-build) for more details.
+See the Releases section of the
+[Enterprise Suite Build and Release Process](https://github.com/lightbend/helm-charts/tree/master/enterprise-suite)
+documentation.
```

Comment:
We should probably pull this, because it's in the public repo. I already had someone random try to access the doc and got an email about it.